bundles/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/WakeOnLanUtility.java

```diff
@@ -102,30 +116,33 @@ public class WakeOnLanUtility {
     /**
      * Send single WOL (Wake On Lan) package on all interfaces
      *
-     * @param macAddress MAC address to send WOL package to
+     * @macAddress MAC address to send WOL package to
```

Comment:
This reverts #15701:
```suggestion
     * @param macAddress MAC address to send WOL package to
```

Reply:
Fixed